gradle-tests/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
-#Mon Aug 21 14:13:24 PDT 2023
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
+networkTimeout=10000
```

Comment:
https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:adding_wrapper

These lines are added/removed automatically by `gradlew wrapper`. Depending on which version you originally used to create them, it might be a different one. I personally always run it twice to ensure the new version's setup is used.